source/1-SDLC-organization/lib/cdk-bootstrap-template.yml

```diff
@@ -0,0 +1,751 @@
+            Version: '2012-10-17'
+          PolicyName: default
+      RoleName:
+        Fn::Sub: cdk-${Qualifier}-deploy-role-${AWS::AccountId}-${AWS::Region}
+  CloudFormationExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: cloudformation.amazonaws.com
+        Version: '2012-10-17'
+      ManagedPolicyArns:
+        Fn::If:
+          - HasCloudFormationExecutionPolicies
+          - Ref: CloudFormationExecutionPolicies
+          - Fn::If:
+              - HasTrustedAccounts
+              # The CLI will prevent this case from occurring
+              - Ref: AWS::NoValue
+              # The CLI will advertise that we picked this implicitly
+              - - Fn::Sub: "arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess"
+      PermissionsBoundary:
+        Ref: CloudFormationExecutionPermissionsBoundary
+      RoleName:
+        Fn::Sub: cdk-${Qualifier}-cfn-exec-role-${AWS::AccountId}-${AWS::Region}
+  CloudFormationExecutionPermissionsBoundary:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - s3:CreateBucket
+              - s3:DeleteBucket
+              - s3:DeleteBucketPolicy
+              - s3:DeleteJobTagging
+              - s3:DeleteObjectTagging
+              - s3:DeleteObjectVersionTagging
+              - s3:PutBucketPolicy
+              - s3:PutBucketPublicAccessBlock
+              - s3:PutBucketTagging
+              - s3:PutJobTagging
+              - s3:PutObjectTagging
+              - s3:PutObjectVersionTagging
+              - s3:GetEncryptionConfiguration
+              - s3:PutEncryptionConfiguration
+            Resource:
+              - "*"
+          - Effect: Allow
+            Action:
+              - codepipeline:CreateCustomActionType
+              - codepipeline:CreatePipeline
+              - codepipeline:DeletePipeline
+              - codepipeline:DeleteWebhook
+              - codepipeline:GetPipeline
+              - codepipeline:GetPipelineState
+              - codepipeline:DeregisterWebhookWithThirdParty
+              - codepipeline:PutWebhook
+              - codepipeline:RegisterWebhookWithThirdParty
+              - codepipeline:TagResource
+              - codepipeline:UntagResource
+              - codepipeline:UpdatePipeline
+            Resource: "*"
+
+            # PipelineSelfMutate
+          - Effect: Allow
+            Action:
+              - ssm:GetParameters
+            Resource: !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/cdk-bootstrap/*"
+
+            # SecretsManagerForGithub
+          - Effect: Allow
+            Action:
+              - secretsmanager:GetSecretValue
+            Resource: !Sub "arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:GITHUB_TOKEN*"
```

Comment:
`GITHUB_TOKEN` is a customer value so should not be restricted based on secret name. I don't think we can do more than `arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:*`

Reply:
Agree. Will change.